tests/CoreCLRTestsNativeArtifacts_OSX.txt

```diff
@@ -0,0 +1 @@
+https://ci.dot.net/job/dotnet_coreclr/job/master/job/debug_osx10.12/lastSuccessfulBuild/artifact/bin/obj/OSX.x64.Debug/tests/*zip*/tests.zip
```

Comment:
You probably don't want the last successful build here. If the artifacts change, it could break us. For the regular test artifacts I pinned a specific build in Jenkins so the artifacts don't get GC'd and we can rely on the stability of a constant set of test binaries.